src/commands/run.php

```diff
@@ -49,6 +49,13 @@
 $codeception_args = array_merge( [ 'run' ], $args( '...' ) );
 ensure_service_running( 'slic' );
 
+// Due to a Docker bug, confirm Linux has proper permissions, otherwise output an error.
+$check_permissions = confirm_linux_has_file_permissions( [ slic_wp_dir(), slic_plugins_dir( slic_target( true ) ) ] );
```

Comment:
Same note here about the one-line function call and exit on failure.

Reply:
While `here` might not be frequently used, `run` will likely be used more often: running this check on all the thousands of file comprising a WordPress site/plugin/theme might have a severe impact on performance.
Can you quantify a before/after on your machine?
To avoid a check, since you're on Linux, you can use the file and dir `mtime` and skip the check if one of the paths you're looking for has its `mtime` unchanged since last check.
You can save the last check information in a dot file somewhere in the `slic` directory; that dot file would be removed on a `slic reset` call.